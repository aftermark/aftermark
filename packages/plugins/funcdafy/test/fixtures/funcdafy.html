<!DOCTYPE html>
<body>
  <h1 class="preexisting-class">Header with nested <span>span</span></h1>
  <p>Paragraph with nested <span>span</span>.</p>
</body>
