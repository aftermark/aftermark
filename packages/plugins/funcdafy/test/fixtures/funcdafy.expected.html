<!DOCTYPE html>
<body>
  <h1 class="preexisting-class header-1 mixin-1 mixin-2">Header with nested <span>span</span></h1>
  <p class="paragraph mixin-1 mixin-2">Paragraph with nested <span class="paragraph-span">span</span>.</p>
</body>
